content/java/concepts/output/terms/print/print.md

```diff
@@ -0,0 +1,45 @@
+---
+Title: '.print()'
+Description: 'Prints its argument to the console.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Output'
+  - 'Strings'
+  - 'Print'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The `.print()` method prints its argument to the console. Unlike the similar method [`.println()`](https://www.codecademy.com/resources/docs/java/output/println), `.print()` does not follow its argument with a new line, and any subsequent characters sent to the console will begin wherever the prior `.print()` command left off.
```

Comment:
From my little time learning Java I believe `.println()` is preferred. I think we should add something like that.

Reply:
I think our goal with Docs is for our content to be as neutral and un-opinionated as possible. There might be cases where printing multiple things on the same line is necessary; other times it might make more sense to print some things/each subsequent thing on a new ling. As I look at this part of the entry (and I'm just now starting my review of it), this phrasing seems fine to me.